Automattic/jetpack
# Markdown: Fix broken links with single-quoted href attributes

#### Changes proposed in this Pull Request:

The changes in https://github.com/Automattic/jetpack/pull/17790 seem to have caused a regression: Running the Markdown parser on links that use a single-quoted `href` attribute (such as `<a href='https://example.com'>example</a>`) mangles the HTML (result: `&lt;a href='https://example.com'>example</a>`).

The fix seems to be as simple as adding single quotes to the list of characters in the regexp added in that pull request.

#### Jetpack product discussion
- N/A

#### Does this pull request change what data or activity we track or use?
- No

#### Testing instructions:

I would have expected to be able to reproduce this bug with a testing setup similar to the one described in https://github.com/Automattic/jetpack/pull/17790, but so far I'm not able to.

However, we ran into this issue in a slightly more complicated scenario on WordPress.com (pNPgK-5no-p2), and I can confirm that this change seems to fix the bug there.
